#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Remove link to pthreads manual
# DP: Author: Jeff Bailey
# DP: Upstream status: Pending
# DP: Status Details: Linuxthreads manual only exists on Linux and BSD.
# DP: Date: 12 Oct 2002

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
--- manual/signal.texi	29 Jan 2002 08:57:47 -0000	1.68
+++ manual/signal.texi	12 Oct 2002 20:23:30 -0000
@@ -2565,7 +2565,7 @@ Note that you must not use @code{sigproc
 because each thread has its own signal mask and there is no single process
 signal mask. According to POSIX, the behavior of @code{sigprocmask} in a
 multi-threaded process is ``unspeficied''. Instead, use 
-@code{pthread_sigmask} (@pxref{Threads and Signal Handling}).
+@code{pthread_sigmask}.

 @comment signal.h
 @comment POSIX.1



